tools/ci_build/build.py

```diff
@@ -358,6 +353,12 @@ def parse_arguments():
     parser.add_argument(
         "--build_micro_benchmarks", action='store_true',
         help="Build ONNXRuntime micro-benchmarks.")
+    parser.add_argument(
+        "--include_ops_by_model", type=str,
+        help="include ops from model(s) under designated path.")
+    parser.add_argument(
+        "--include_ops_by_file", type=str,
+        help="include ops from csv file.")
```

Comment:
It's no longer csv, right?

Reply:
Will get it right finally when we have agreement by discussion.

Reply:
Taking foramt:

domain;opset;op1,op2,...

Reply:
Done.